sample-android-app/build.gradle

```diff
@@ -5,12 +5,12 @@ plugins {
 
 android {
     namespace "com.badoo.reaktive.sample.android"
-    compileSdk 31
+    compileSdk 33
```

Comment:
34 maybe?

Reply:
I don't mind, updated. 33 was the minimum required.